src/tabset/tabset.spec.ts

```diff
@@ -122,6 +122,7 @@ describe('ngb-tabset', () => {
 
     expect(tabTitles[0].textContent).toMatch(/foo/);
     expect(tabTitles[1].innerHTML).toMatch(/<b>bar<\/b>/);
+    expect(tabTitles[1].classList.contains('barClass')).toBe(true);
```

Comment:
I'll make that update.  I'm not familiar with the jasmine unit testing framework, so just went with the easy "for dummies" solution.